crates/viewer/re_space_view_dataframe/src/space_view_class.rs

```diff
@@ -52,9 +74,8 @@ In the _time range_ mode, the view displays all the data logged within the time
 view entity. In this mode, each row corresponds to an entity and time pair. Rows are further split
 if multiple `rr.log()` calls were made for the same entity/time. Static data is also displayed.
 
-The view switches to _time range_ mode as soon as a single one of its entities has its visible time
-range set to _Override_. Each entity may have its own time range setting. (To set the same time range
-for all entities, it is preferable to override the view-level visible time range.)"
+Note that the default visible time range depends on the selected mode. In particular, the time range
+mode sets the default time range to _everything_. You can override this in the selection panel."
```

Comment:
But what does setting the query to visible time range do when I'm in `LatestAt` mode? I tried and it doesn't seem to do anything.

Reply:
For now, it does nothing. The "Latest at" mode implies "Latest at query as per the global time cursor", and overrides whatever might be set as "Visible Time Range".

Not ideal. And it's gonna get worse: I'll soon need a "Latest at t= *specific absolute time*". I might use the Visible time range setting in a very hacky way for that. IMO, this is exactly the kind of conceptual/design mess that we must sort out for the dataframe view to go out of "beta".